src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -1366,6 +1367,61 @@ bool FancyZones::ShouldProcessSnapHotkey(DWORD vkCode) noexcept
     return false;
 }
 
+void FancyZones::ApplyQuickLayout(int vKey) noexcept

```

Comment:
What about moving this method to a dedicated file, since we already know it will grow in complexity when we will add support for templates?
Let's try to abstract it as much as possible from the FancyZones class,

Reply:
That's a good idea but I'm not sure how to do it at the moment.